services/app-web/src/testHelpers/apolloHelpers.tsx

```diff
@@ -836,25 +852,94 @@ const mockSubmittedHealthPlanPackageWithRevision = (): HealthPlanPackage => {
     }
 }
 
-type updateHealthPlanFormDataMockSuccessProps = {
+type updateHealthPlanFormDataMockProps = {
     pkg?: HealthPlanPackage
     updatedFormData: string
     id: string
+    statusCode?: 200 | 403 | 404 | 500
 }
 
-const updateHealthPlanFormDataMockSuccess = ({
+const updateHealthPlanFormDataMock = ({
     pkg = mockUnlockedHealthPlanPackage(),
     updatedFormData,
     id,
-}: updateHealthPlanFormDataMockSuccessProps): MockedResponse<UpdateHealthPlanFormDataMutation> => {
-    return {
-        request: {
-            query: UpdateHealthPlanFormDataDocument,
-            variables: {
-                input: { pkgID: id, healthPlanFormData: updatedFormData },
-            },
-        },
-        result: { data: { updateHealthPlanFormData: { pkg } } },
+    statusCode,
+}: updateHealthPlanFormDataMockProps): MockedResponse<UpdateHealthPlanFormDataMutation> => {
+    switch (statusCode) {
+        case 200:
+            return {
+                request: {
+                    query: UpdateHealthPlanFormDataDocument,
+                    variables: {
+                        input: {
+                            pkgID: id,
+                            healthPlanFormData: updatedFormData,
+                        },
+                    },
+                },
+                result: { data: { updateHealthPlanFormData: { pkg } } },
+            }
+        case 403:
+            return {
+                request: { query: UpdateHealthPlanFormDataDocument },
```

Comment:
why does this not have any input passed in? Isn’t that required for it to match?

Reply:
maybe, or maybe we aren’t actually calling it. Another good reason to break this out from the switch statement, it will be obvious if that function is being called or not

Reply:
I split the helpers I consolidated into into individual ones.